packages/devtools_app/lib/src/inspector/flutter/inspector_screen_details_tab.dart

```diff
@@ -103,16 +101,64 @@ class _LayoutDetailsTabState extends State<LayoutDetailsTab>
 
   RemoteDiagnosticsNode get selected => controller?.selectedNode?.diagnostic;
 
-  void onSelectionChanged() {
+  InspectorObjectGroupManager objectGroupManager;
+
+  RemoteDiagnosticsNode root;
+
+  RemoteDiagnosticsNode getRoot(RemoteDiagnosticsNode node) {
+    if (!StoryOfYourFlexWidget.shouldDisplay(node)) return null;
+    if (node.isFlex) return node;
+    return node.parent;
```

Comment:
only return the parent if the parent is flex and the node is not flex.

Reply:
yes the case should be handled in the `StoryOfYourFlexWidget.shouldDisplay(node)`